SPECS-EXTENDED/wireshark/wireshark.spec

```diff
@@ -1,90 +1,68 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
 %global with_lua 1
 %global with_maxminddb 1
 %global plugins_version 3.4
-%global with_gui 0
 
-Summary:	Network traffic analyzer
-Name:		wireshark
-Version:	3.4.4
-Release:	5%{?dist}
-License:	BSD and GPLv2
-Url:		http://www.wireshark.org/
-
-Source0:	https://wireshark.org/download/src/all-versions/%{name}-%{version}.tar.xz
-Source1:        https://www.wireshark.org/download/src/all-versions/SIGNATURES-%{version}.txt
-Source2:	90-wireshark-usbmon.rules
+Summary:	      Network traffic analyzer
+Name:		        wireshark
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Version:	      3.4.14
+Release:	      1%{?dist}
+License:	      BSD and GPLv2
+Url:		        http://www.wireshark.org/
+Source0:	      https://2.na.dl.wireshark.org/src/all-versions/%{name}-%{version}.tar.xz 
+Source1:	      90-wireshark-usbmon.rules
 
 # Fedora-specific
```

Comment:
We might not need the "Fedora-specific" patches. Checking that is potentially out of scope of this PR, though.

Reply:
yeah..out of scope this time.

Reply:
True, but when we were doing a global `fedora` macros clean-up, we decided to keep these parts of the spec, since we build them in a very similar way to Fedora. If we're not able to build correctly with these spec sections, then I'm fine removing them but otherwise it is usually easier to integrate with other Fedora-derived packages (in case of dependencies) if we keep these sections.